released/packages/coq-vst-32/coq-vst-32.2.9.1/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+synopsis: "Verified Software Toolchain"
+description: "The software toolchain includes static analyzers to check assertions about your program; optimizing compilers to translate your program to machine language; operating systems and libraries to supply context for your program. The Verified Software Toolchain project assures with machine-checked proofs that the assertions claimed at the top of the toolchain really hold in the machine-language program, running in the operating-system context."
+authors: [
+  "Andrew W. Appel"
+  "Lennart Beringer"
+  "Sandrine Blazy"
+  "Qinxiang Cao"
+  "Santiago Cuellar"
+  "Robert Dockins"
+  "Josiah Dodds"
+  "Nick Giannarakis"
+  "Samuel Gruetter"
+  "Aquinas Hobor"
+  "Jean-Marie Madiot"
+  "William Mansky"
+]
+maintainer: "VST team"
+homepage: "http://vst.cs.princeton.edu/"
+dev-repo: "git+https://github.com/PrincetonUniversity/VST.git"
+bug-reports: "https://github.com/PrincetonUniversity/VST/issues"
+
+# ATTENTION: the source package also contains software with different license,
+# but what this opam package installs is BSD-2-Clause. In case the opam build,
+# test or install commands are changed, this needs to be reviewed!
+license: "BSD-2-Clause"
+
+build: [
+  [make "-j%{jobs}%" "vst" "IGNORECOQVERSION=true" "BITSIZE=32"]
+]
+install: [
+  [make "install" "IGNORECOQVERSION=true" "BITSIZE=32"]
+]
+run-test: [
+  [make "-j%{jobs}%" "test" "IGNORECOQVERSION=true" "BITSIZE=32"]
+]
+depends: [
+  "ocaml"
+  "coq" {>= "8.12" & < "8.16~"}
+  "coq-compcert-32" {= "3.10"}
+  "coq-flocq" {>= "3.2.1"}
```

Comment:
```suggestion
  "coq-flocq" {>= "3.2.1" & < "4~"}
```
Since CompCert 3.10 only supports Flocq `< "4~"`, omitting the upper bound here is a bit misleading (same in other package).

Reply:
VST makes only little use of Flocq itself - there are some proofs about float expressions, but I don't think there is a single Require Flocq statement in VST, so I would more think about removing this dependency.

Afair there are a few float examples, but as far as I can see they also don't require Flocq directly.